cdap-common/bin/functions.sh

```diff
@@ -643,6 +643,14 @@ cdap_start_java() {
       __defines+=" -Djava.library.path=${JAVA_LIBRARY_PATH}"
     fi
     __startup_checks=${CDAP_STARTUP_CHECKS:-$(cdap_get_conf "master.startup.checks.enabled" "${CDAP_CONF}"/cdap-site.xml true)}
+
+    # Build and upload coprocessor jars
+    logecho "$(date) Ensuring required HBase coprocessors are on HDFS"
+    "${JAVA}" ${JAVA_HEAPMAX} ${OPTS} -cp ${CLASSPATH} co.cask.cdap.data.tools.CoprocessorBuildTool </dev/null >>${__logfile} 2>&1
```

Comment:
the problem is it would re-run all the functions that set classpath and everything.